### PR TITLE
Add delete group member functionality to the module

### PR DESCRIPTION
**Pull request by dkorn**

### CODE REVIEW

**Comment by dkorn** [comment]
@cben @enoodle @kwoodson @ivanhorvath @simon3z please review

File: library/hawkular_alerts_group_member.py [hunk 1]
```diff
@@ -48,8 +48,10 @@
   state:
     description:
       - the state of the user
-      - On present, it will create the group trigger
+      - On present, it will create the group member trigger
       if it does not exist
+      - On absent, it will delet the group member trigger
```
Comment (cben) [inline library/hawkular_alerts_group_member.py]: delete

File: library/hawkular_alerts_group_member.py [hunk 2]
```diff
@@ -120,7 +122,25 @@ def group_member_exist(self, group_id, name):
             group_members = self.client.get_group_members(group_id)
         except Exception as e:
             self.module.fail_json(msg="Failed to get group members. Error: {error}".format(error=e))
-        return next((True for gm in group_members if gm.name == name), False)
+        return next((True for gm in group_members if gm.id == member_id), False)
```
Comment (cben) [inline library/hawkular_alerts_group_member.py]: btw, what do you think of `any(gm.id == member_id for gm in group_members)` ?
perhaps more readable than `next`with booleans

Comment (cben) [inline library/hawkular_alerts_group_member.py]: another q: if now you're looking by id, doesn't the client offer an efficient way to access only that member?

**Comment by cben** [comment]
Do you consistently look by id vs name?
Is it mentioned anywhere in doc?

File: library/hawkular_alerts_group_member.py (hunk 1 above)
Comment (dkorn) [reply library/hawkular_alerts_group_member.py]: done, thanks

File: library/hawkular_alerts_group_member.py (hunk 2 above)
Comment (dkorn) [reply library/hawkular_alerts_group_member.py]: I agree, will make the change thanks!

Comment (dkorn) [reply library/hawkular_alerts_group_member.py]: I can try to get the specific group member trigger by id and if I get 404 error return False.

**Comment by dkorn** [comment]
> Do you consistently look by id vs name?

Haven't decided yet, it may be more convenient for users to query by name

> Is it mentioned anywhere in doc?

No docs currently :/

**Comment by cben** [comment]
LGTM

**Pull request merged**